src/components/ContentListTable/components/AddContent/AddContent.tsx

```diff
@@ -575,7 +575,17 @@ const AddContent = ({ isLoading }: Props) => {
                               if (isValidURL(value)) {
                                 updateVariable(index, { gpgLoading: true });
                                 const gpgData = await fetchGpgKey(value);
-                                updateVariable(index, { gpgKey: gpgData, gpgLoading: false });
+                                updateVariable(index, {
+                                  gpgKey: gpgData,
+                                  gpgLoading: false,
+                                  ...(gpgKey === '' && !!value
+                                    ? {
+                                        metadataVerification:
+                                          !!validationList?.[index]?.url
+                                            ?.metadata_signature_present,
+                                      }
+                                    : {}),
+                                });
```

Comment:
is this change a remnant of some other PR? it doesn't strike me as a "restructure".

Reply:
hmm good catch. Ugh.. this needs to be removed.

Reply:
Actually. going to check. I'm concerned that the onTextChange below may not work without this, and that's why I added it here. Testing now.